docs/open-api/permissions.yaml

```diff
@@ -207,23 +207,108 @@ paths:
         "5xx":
           $ref: "./openapi.yaml#/components/responses/ServerErrorResponse"
 
+  /metalakes/{metalake}/permissions/roles/{role}/{metadataObjectType}/{metadataObjectFullName}/grant:
+    parameters:
+      - $ref: "./openapi.yaml#/components/parameters/metalake"
+      - $ref: "./openapi.yaml#/components/parameters/role"
+      - $ref: "./openapi.yaml#/components/parameters/metadataObjectType"
+      - $ref: "./openapi.yaml#/components/parameters/metadataObjectFullName"
+
+    put:
+      tags:
+        - access control
+      summary: Grant privileges to a role
+      operationId: grantPrivilegeToRole
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/PrivilegeGrantRequest"
+            examples:
+              RoleRevokeRequest:
+                $ref: "#/components/examples/PrivilegeGrantRequest"
```

Comment:
“RoleRevokeRequest” I think it is not correct, right?

Reply:
Corrected.